keepmenu

```diff
@@ -53,8 +53,8 @@ def random_str():
     return ''.join(random.choice(letters) for i in range(15))
 
 
-def gen_passwd(length=20, use_digits=True, use_spec_chars=True):
-    """Generate password (min 4 chars, 1 lower, 1 upper, 1 digit, 1 special char)
+def gen_passwd(length=20, alphabet_list=[]):
+    """Generate password (min 4 chars or list length, at least one used of each group)
```

Comment:
Why is the default `alphabet_list=[]`? Seems like that shouldn't have a default value or should be set to whatever the default value is instead of a blank list. Also, please update the doc string for that function to reflect the new arguments/return values.